crates/sui-indexer-alt-schema/migrations/2024-12-05-162243_add_coin_balance_owner_kind_indexes/down.sql

```diff
@@ -0,0 +1,2 @@
+CREATE INDEX CONCURRENTLY IF NOT EXISTS wal_coin_balances_owner_type
+ON wal_coin_balances (owner_id, coin_type, coin_balance, object_id, object_version);
```

Comment:
thoughts on doing this directly instead of through a migration, or are we too late to do that

Reply:
So this is not strictly to do with this PR, but the plan was to do both -- i.e. land it, and also run it proactively, to avoid it stalling the deploy. It ended up taking over a day just to land the first migration though, so that's why this stuff got backed out!

In future, we would probably do something like the following, if a PR includes non-trivial migrations:

- Get the PR accepted.
- Run the migrations while the PR is accepted.
- Land the PR once the migrations have completed.

This makes sure we don't hold up other deploys or introduce a long lead time when the service has been restarted.